### ISSUE TITLE
skid af stop selling free stuff that u pasted

### ISSUE THREAD

**Issue by Syntheticc** [KSCHdsc/BlackCap-Grabber#8]
please make your own grabber stop pasting doesn't matter if u have credits your still selling it... and its free

**Comment by KSCHdsc** [comment]
Then Why did you steal my injection in your github :))

**Comment by Syntheticc** [comment]
> Then Why did you steal my injection in your github :))

first it isnt your injection, second im not selling that crap it barely works, and i js hosting to see if it works and it didnt causes errors in discord

**Comment by KSCHdsc** [comment]
Yes, it's my injection. i got fork log :)
and secondly I don't sell the code, I sell the builder which makes it undetectable

**Comment by Syntheticc** [comment]
> Yes, it's my injection. i got fork log :) and secondly I don't sell the code, I sell the builder which makes it undetectable

u pasted the injection it doesnt even work too u literally are selling it

**Comment by KSCHdsc** [comment]
the injection works very well and I dev it myself even if you are so sure that it finds me the original repo

**Comment by Syntheticc** [comment]
> the injection works very well and I dev it myself even if you are so sure that it finds me the original repo

"- Based on PirateStealer injection and Hazard Nuker injection "

**Comment by KSCHdsc** [comment]
yes but not the same module not the same embed not the same request not the same variable not the same function i just take the syntax you can verify. :(
Now, let me make my code and don't spam issue in this repo.

**Comment by Syntheticc** [comment]
> yes but not the same module not the same embed not the same request not the same variable not the same function i just take the syntax you can verify. :( Now, let me make my code and don't spam issue in this 

stop pasting

**Comment by KSCHdsc** [comment]
okay you're right that's why 35 people liked my project I'm definitely blocking you from now on
